# The minimum turning radius of the vehicle
r: 0.5
# deltat: 0.706
deltat: 0.706
# A movement cost penalty for turning (choosing non straight motion primitives)
penaltyTurning: 3.0
# penaltyTurning: 10.0
# A movement cost penalty for reversing (choosing motion primitives >
# 2)
penaltyReversing: 0.5
# A movement cost penalty for change of direction (changing from primitives < 3 to primitives > 2)
penaltyCOD: 2.0
# map resolution
mapResolution: 2.0

# width of car
carWidth: 3
# distance from rear to vehicle front end
LF: 1.5
# distance from rear to vehicle back end
LB: 1.5
# obstacle default radius
obsRadius: 0.1
# least time to wait for constraint
constraintWaitTime: 1.0
